build/build.js

```diff
@@ -63,11 +65,17 @@ catchphraseButton.addEventListener('click', async() => {
 });
 
 window.addEventListener('load', async() => {
-    let character;
+    let character = await getCharacter;
     // on load, attempt to fetch this user's character
 
     // if this user turns out not to have a character
-    // create a new character with correct defaults for all properties (head, middle, bottom, catchphrases)
+    if (!character) {
+        // create a new character with correct defaults for all properties (head, middle, bottom, catchphrases)
+        const newCharacter = await createDefaultCharacter();
```

Comment:
I think this is just a typo.  The function you are importing is called `createCharacter`